akka-persistence-typed/src/test/scala/akka/persistence/typed/scaladsl/EventSourcedBehaviorWatchSpec.scala

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.typed.scaladsl
+
+import java.util.concurrent.atomic.AtomicInteger
+
+import akka.actor.testkit.typed.TestException
+import akka.actor.testkit.typed.scaladsl.{ LogCapturing, LoggingTestKit, ScalaTestWithActorTestKit, TestProbe }
+import akka.actor.typed.scaladsl.Behaviors
+import akka.actor.typed.{ ActorRef, Behavior, ChildFailed, DeathPactException, Terminated }
+import akka.persistence.typed.PersistenceId
+import akka.serialization.jackson.CborSerializable
+import org.scalatest.WordSpecLike
+
+import scala.util.Random
+
+object EventSourcedBehaviorWatchSpec {
+  sealed trait Command extends CborSerializable
+  case object Fail extends Command
+  case object Stop extends Command
+  final case class ChildHasFailed(t: akka.actor.typed.ChildFailed)
+  final case class HasTerminated(ref: ActorRef[_])
+}
+
+class EventSourcedBehaviorWatchSpec
+    extends ScalaTestWithActorTestKit(EventSourcedBehaviorSpec.conf)
+    with WordSpecLike
+    with LogCapturing {
+
+  import EventSourcedBehaviorWatchSpec._
+
+  private val cause = TestException("Dodge this.")
+
+  private val parentId = new Random()
+
+  private val pidCounter = new AtomicInteger(0)
+
+  private def nextPid: PersistenceId = PersistenceId.ofUniqueId(s"${pidCounter.incrementAndGet()}")
+
+  private def spawnAnonymous[T](behavior: Behavior[T]): ActorRef[T] = {
+    spawn(behavior, name = s"${parentId.nextInt()}")
```

Comment:
`spawn` in the testkit doesn't require a `name` and will generate a unique name if not specified